mobile-widgets/qmlprefs.cpp

```diff
@@ -0,0 +1,197 @@
+// SPDX-License-Identifier: GPL-2.0
+#include "qmlprefs.h"
+#include "qmlmanager.h"
+
+#include "core/membuffer.h"
+#include "core/subsurface-qt/SettingsObjectWrapper.h"
+#include "core/gpslocation.h"
+
+
+/*** Global and constructors ***/
+QMLPrefs *QMLPrefs::m_instance = NULL;
+
+QMLPrefs::QMLPrefs() :
+	m_cloudPassword(""),
```

Comment:
QString() default constructs to the empty string. This line and the next two are not needed.

Reply:
Corrected (sorry this was just writing without my head somewhere else)